docker/dnf/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:latest
+
+WORKDIR /ahab-docker
+
+RUN dnf upgrade -y && dnf check-update
+
+COPY ahab .
+
+# Spit out these just for easier debugging
+RUN dnf list installed
+
+RUN dnf list installed | ./ahab chase --os fedora
```

Comment:
Do you even need the OS now?

Reply:
Correct it's not required but I have a dockerfile that tests if you specify it and not. Thus this file

Reply:
Gotcha!